App.js

```diff
@@ -289,12 +249,55 @@ export default function App() {
   return (
     <View style={styles.container}>
       <View style={styles.nav}>
-        <FlatList data={nav} renderItem={renderNav} style={styles.nav} numColumns={numColumns} />
+        <Button
+          onPress={() => pressReset()}
+          title="RESET"
+          color="blue"
+        />
+        <View
+          style={
+            (message === 'Game Complete') ? styles.itemInvisible : (
+            even ? [styles.item, styles.markSpot1] : [styles.item, styles.itemBlue] )
+          }
+        >
+        <TouchableHighlight onPress={() => pressEven()}>
```

Comment:
You couldddd make a new component for the the odd/even button to share since this piece of JSX is the same. It's a pretty small, but you use this pattern all over the place aha

Reply:
Do you mean a formal separate component or an externalizing of the code like I did the board?

Reply:
I get what you mean.....but I don't use it as much any more......as I converted the PLAY (now called RESET) and ROLL buttons in 'Buttons' now.